src/graphql/queries/total-result-bets.js

```diff
@@ -27,19 +27,40 @@ module.exports = async (
   if (isNull(event)) throw Error('Event not found');
   const { numOfResults } = event;
 
+  // Accumulate all round 0 bets
+  let totalBetRound = await DBHelper.findBet({ eventAddress, eventRound: 0 });
+  const totalBets = accumulateBets(numOfResults, totalBetRound);
+
+  // Accumulate better round 0 bets
+  let betterBets
+  if (betterAddress) {
+    betterBetRound = await DBHelper.findBet({ eventAddress, eventRound: 0, betterAddress });
```

Comment:
whats the diff between this and `roundBets` in the event query? seems the same as that except that its all the votes instead of just the current round.

Reply:
its the same as for the betting round, but roundBets can get other round